lib/vtls/wolfssl.c

```diff
@@ -202,6 +202,43 @@ static int do_file_type(const char *type)
   return -1;
 }
 
+#ifdef HAVE_LIBOQS
+struct group_name_map {
+  const word16 group;
+  const char   *name;
+};
+
+static const struct group_name_map gnm[] = {
+  { .group = WOLFSSL_KYBER512, .name = "KYBER512" },
+  { .group = WOLFSSL_KYBER768, .name = "KYBER768" },
+  { .group = WOLFSSL_KYBER1024, .name = "KYBER1024" },
+  { .group = WOLFSSL_NTRU_HPS2048509, .name = "NTRU_HPS2048509" },
+  { .group = WOLFSSL_NTRU_HPS2048677, .name = "NTRU_HPS2048677" },
+  { .group = WOLFSSL_NTRU_HPS4096821, .name = "NTRU_HPS4096821" },
+  { .group = WOLFSSL_NTRU_HRSS701, .name = "NTRU_HRSS701" },
+  { .group = WOLFSSL_LIGHTSABER, .name = "LIGHTSABER" },
+  { .group = WOLFSSL_SABER, .name = "SABER" },
+  { .group = WOLFSSL_FIRESABER, .name = "FIRESABER" },
+  { .group = WOLFSSL_KYBER90S512, .name = "KYBER90S512" },
+  { .group = WOLFSSL_KYBER90S768, .name = "KYBER90S768" },
+  { .group = WOLFSSL_KYBER90S1024, .name = "KYBER90S1024" },
+  { .group = WOLFSSL_P256_NTRU_HPS2048509, .name = "P256_NTRU_HPS2048509" },
+  { .group = WOLFSSL_P384_NTRU_HPS2048677, .name = "P384_NTRU_HPS2048677" },
+  { .group = WOLFSSL_P521_NTRU_HPS4096821, .name = "P521_NTRU_HPS4096821" },
+  { .group = WOLFSSL_P384_NTRU_HRSS701, .name = "P384_NTRU_HRSS701" },
+  { .group = WOLFSSL_P256_LIGHTSABER, .name = "P256_LIGHTSABER" },
+  { .group = WOLFSSL_P384_SABER, .name = "P384_SABER" },
+  { .group = WOLFSSL_P521_FIRESABER, .name = "P521_FIRESABER" },
+  { .group = WOLFSSL_P256_KYBER512, .name = "P256_KYBER512" },
+  { .group = WOLFSSL_P384_KYBER768, .name = "P384_KYBER768" },
+  { .group = WOLFSSL_P521_KYBER1024, .name = "P521_KYBER1024" },
+  { .group = WOLFSSL_P256_KYBER90S512, .name = "P256_KYBER90S512" },
+  { .group = WOLFSSL_P384_KYBER90S768, .name = "P384_KYBER90S768" },
+  { .group = WOLFSSL_P521_KYBER90S1024, .name = "P521_KYBER90S1024" },
+  { .group = 0, .name = NULL }
```

Comment:
We strive for C89 compatibility, and while perhaps nobody is actually going to build with wolfSSL using such a compiler I fear this might still trigger some build warning/lint checker down the road. If you just remove the `name =` parts of the initialization here I figure it won't be needing C99 anymore.

Reply:
Oh...ok.  I can do that. Didn't realize there was a preference for C89.  I'll try to keep that in mind for the future.